fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -145,6 +145,17 @@ ThreadFuture<Standalone<StringRef>> DLTransaction::getVersionstamp() {
 	});
 }
 
+ThreadFuture<int64_t> DLTransaction::getStorageByteSample(const KeyRangeRef& keys) {
+	FdbCApi::FDBFuture *f = api->transactionGetStorageByteSample(tr, keys.begin.begin(), keys.begin.size(), keys.end.begin(), keys.end.size());
```

Comment:
This will need to fail if the function doesn't exist in the external library. See `getApproximateSize` for an example.

Reply:
Thanks!